<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget_settings</class>
 <widget class="QWidget" name="widget_settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>448</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>448</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#widget_settings
{
	background: #FFFFFF;
	border-top-left-radius: 6px;
	border-top-right-radius: 6px;
}</string>
  </property>
  <widget class="QWidget" name="title_widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>448</width>
     <height>48</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#title_widget
{
	background:#00D770;
	border-top-left-radius: 6px;
	border-top-right-radius: 6px;
}
</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>16</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/logo.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_title">
    <property name="geometry">
     <rect>
      <x>53</x>
      <y>16</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_title
{
	color: #FFFFFF;
	font-size: 14px;
}</string>
    </property>
    <property name="text">
     <string>设置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_close">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>16</y>
      <width>22</width>
      <height>22</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;关闭&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">#button_close
{
	border-image: url(:/image/close.png);
}

#button_close:hover
{
	background: green;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_bottom" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>448</width>
     <height>600</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>448</width>
     <height>600</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>448</width>
     <height>600</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">#widget_bottom
{
	background: #FFFFFF;
	border-top-left-radius: 6px;
	border-top-right-radius: 6px;
	border: 1px solid gray;
}</string>
   </property>
   <widget class="QLabel" name="label_udp_port">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>280</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_udp_port
{
	color: #47505D;
	font-size: 13px;
}</string>
    </property>
    <property name="text">
     <string>UDP端口:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineedit_gateway">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>220</y>
      <width>331</width>
      <height>36</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">#lineedit_gateway
{
	border-radius: 4px;
	background: #F0F4F5;
	selection-color: red;
	padding-left: 10px;
}

#lineedit_gateway:hover
{
	border: 1px solid #00D770;
}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineedit_tcp_port">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>270</y>
      <width>120</width>
      <height>36</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">#lineedit_tcp_port
{
	border-radius: 4px;
	background: #F0F4F5;
	selection-color: red;
	padding-left: 10px;
}

#lineedit_tcp_port:hover
{
	border: 1px solid #00D770;
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_cancel">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>550</y>
      <width>110</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#button_cancel
{
	background: #FFFFFF;
	border-radius: 4px;
	border-color: #00D770;
	border-width: 1px;
	border-style: solid;

	font-size: 13px;
	color: #00D770;
}
</string>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_gateway">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_gateway
{
	color: #47505D;
	font-size: 13px;
}</string>
    </property>
    <property name="text">
     <string>网   关:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_ok">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>550</y>
      <width>110</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#button_ok
{
	background: #00D770;
	border-radius: 4px;

	font-size: 13px;
	color: #FFFFFF;
}

#button_ok:hover
{
	background: #00D770
}</string>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tablewidget_share_usb">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>401</width>
      <height>171</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <column>
     <property name="text">
      <string>ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Device</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Share</string>
     </property>
    </column>
   </widget>
   <widget class="QCheckBox" name="checkbox_screen">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>180</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#checkbox_screen
{
	color: #47505D;
	font-size: 13px;
}</string>
    </property>
    <property name="text">
     <string>是否全屏</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineedit_udp_port">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>270</y>
      <width>120</width>
      <height>36</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">#lineedit_udp_port
{
	border-radius: 4px;
	background: #F0F4F5;
	selection-color: red;
	padding-left: 10px;
}

#lineedit_udp_port:hover
{
	border: 1px solid #00D770;
}</string>
    </property>
   </widget>
   <widget class="QComboBox" name="combobox_bitrate">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>130</y>
      <width>120</width>
      <height>36</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">#combobox_bitrate
{
	border-radius: 4px;
	background: #F0F4F5;
	selection-color: red;
}

#combobox_bitrate:hover
{
	border: 1px solid #00D770;
}

QComboBox#combobox_bitrate::drop-down
{
	border-bottom-right-radius: 4px;
	border-top-right-radius: 4px;
}


QComboBox#combobox_bitrate::down-arrow
{
	background: transparent;
	padding: 0px 0px 0px 0px;
	image: url(:/image/icon_down-arrow.png);
}

/*下拉框颜色*/
QComboBox QAbstractItemView
{
	outline: 0px solid gray; /*选定项的虚框*/
	background: #FFFFFF;
	border-bottom-left-radius: 4px;
	border-bottom-right-radius: 4px;
}

QComboBox QAbstractItemView::Item
{
	height: 36px;
}

QComboBox QAbstractItemView::Item:hover
{
	color: white;
	background-color: #00D770;
}

QComboBox QAbstractItemView::Item:selected
{
	color: white;
	background-color: #00D770;
}</string>
    </property>
    <item>
     <property name="text">
      <string>10000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>20000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>30000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>40000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>50000</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_resolution">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_resolution
{
	color: #47505D;
	font-size: 13px;
}</string>
    </property>
    <property name="text">
     <string>分辨率：</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkbox_use_gateway">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#checkbox_use_gateway
{
	color: #47505D;
	font-size: 13px;
}</string>
    </property>
    <property name="text">
     <string>是否使用网关</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="combobox_fps">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>130</y>
      <width>120</width>
      <height>36</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">#combobox_fps
{
	border-radius: 4px;
	background: #F0F4F5;
	selection-color: red;
}

#combobox_fps:hover
{
	border: 1px solid #00D770;
}

QComboBox#combobox_fps::drop-down
{
	border-bottom-right-radius: 4px;
	border-top-right-radius: 4px;
}


QComboBox#combobox_fps::down-arrow
{
	background: transparent;
	padding: 0px 0px 0px 0px;
	image: url(:/image/icon_down-arrow.png);
}

/*下拉框颜色*/
QComboBox QAbstractItemView
{
	outline: 0px solid gray; /*选定项的虚框*/
	background: #FFFFFF;
	border-bottom-left-radius: 4px;
	border-bottom-right-radius: 4px;
}

QComboBox QAbstractItemView::Item
{
	height: 36px;
}

QComboBox QAbstractItemView::Item:hover
{
	color: white;
	background-color: #00D770;
}

QComboBox QAbstractItemView::Item:selected
{
	color: white;
	background-color: #00D770;
}</string>
    </property>
    <item>
     <property name="text">
      <string>30</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>60</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_bitrate">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_bitrate
{
	color: #47505D;
	font-size: 13px;
}</string>
    </property>
    <property name="text">
     <string>比特流：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_fps">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>140</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_fps
{
	color: #47505D;
	font-size: 13px;
}</string>
    </property>
    <property name="text">
     <string>FPS:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="combobox_resolution">
    <property name="geometry">
     <rect>
      <x>88</x>
      <y>80</y>
      <width>331</width>
      <height>36</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">#combobox_resolution
{
	border-radius: 4px;
	background: #F0F4F5;
	selection-color: red;
}

#combobox_resolution:hover
{
	border: 1px solid #00D770;
}

QComboBox#combobox_resolution::drop-down
{
	border-bottom-right-radius: 4px;
	border-top-right-radius: 4px;
}


QComboBox#combobox_resolution::down-arrow
{
	background: transparent;
	padding: 0px 0px 0px 0px;
	image: url(:/image/icon_down-arrow.png);
}

/*下拉框颜色*/
QComboBox QAbstractItemView
{
	outline: 0px solid gray; /*选定项的虚框*/
	background: #FFFFFF;
	border-bottom-left-radius: 4px;
	border-bottom-right-radius: 4px;
}

QComboBox QAbstractItemView::Item
{
	height: 36px;
}

QComboBox QAbstractItemView::Item:hover
{
	color: white;
	background-color: #00D770;
}

QComboBox QAbstractItemView::Item:selected
{
	color: white;
	background-color: #00D770;
}</string>
    </property>
    <item>
     <property name="text">
      <string>1920x1080</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7680x4320</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5120x2880</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4096x2160</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3840x2160</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2560x1600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2560x1440</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2048x1536</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1920x1200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1600x1200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1280x1024</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1280x960</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1280x768</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1024x768</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>800x600</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_share_usb">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_share_usb
{
	color: #47505D;
	font-size: 13px;
}</string>
    </property>
    <property name="text">
     <string>USB 设备</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_tcp_port">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_tcp_port
{
	color: #47505D;
	font-size: 13px;
}</string>
    </property>
    <property name="text">
     <string>TCP端口:</string>
    </property>
   </widget>
  </widget>
  <zorder>widget_bottom</zorder>
  <zorder>title_widget</zorder>
 </widget>
 <tabstops>
  <tabstop>combobox_resolution</tabstop>
  <tabstop>combobox_bitrate</tabstop>
  <tabstop>combobox_fps</tabstop>
  <tabstop>checkbox_use_gateway</tabstop>
  <tabstop>checkbox_screen</tabstop>
  <tabstop>lineedit_gateway</tabstop>
  <tabstop>lineedit_tcp_port</tabstop>
  <tabstop>lineedit_udp_port</tabstop>
  <tabstop>button_ok</tabstop>
  <tabstop>button_cancel</tabstop>
  <tabstop>tablewidget_share_usb</tabstop>
  <tabstop>button_close</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
